packages-content-model/roosterjs-content-model-editor/lib/publicApi/table/applyTableBorderFormat.ts

```diff
@@ -64,56 +83,182 @@ export default function applyTableBorderFormat(
             }
 
             if (sel) {
-                switch (operation) {
-                    case 'AllBorders':
-                        for (let rowIndex = sel.firstRow; rowIndex <= sel.lastRow; rowIndex++) {
-                            for (let colIndex = sel.firstCol; colIndex <= sel.lastCol; colIndex++) {
-                                const cell = tableModel.rows[rowIndex].cells[colIndex];
-                                // Format cells - All borders
-                                applyAllBorderFormat(cell, borderFormat);
+                const operations = [operation];
```

Comment:
Suggest to add a type declaration:

const operatons: BorderOperations[] = [operation];